hr_timesheet_sheet_week_start_day/views/res_company_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="hr_timesheet_sheet_company" model="ir.ui.view">
+            <field name="name">res.company.sheet</field>
+            <field name="model">res.company</field>
+            <field name="inherit_id"
+                   ref="hr_timesheet_sheet.hr_timesheet_sheet_company"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='timesheet_range']"
```

Comment:
you can simplify that to


<field name="timesheet_range" position="after">...</field>